tests/e2e/hcp_machine_pool_test.go

```diff
@@ -114,6 +114,50 @@ var _ = Describe("HCP Machine Pool", labels.Feature.Machinepool, func() {
 		})
 	})
 
+	DescribeTable("create machinepool with volume size set - [id:66872]",
+		labels.Runtime.Day2,
+		labels.Critical,
+		func(diskSize, instanceType, expectedDiskSize string) {
+			npID := common.GenerateRandomName("np-66872", 2)
+
+			By("Create a nodepool with the disk size")
+			output, err := machinePoolService.CreateMachinePool(clusterID, npID,
+				"--replicas", "0",
+				"--disk-size", diskSize,
+				"--instance-type", instanceType,
+			)
+			Expect(err).ToNot(HaveOccurred())
+			textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
+			Expect(textData).
+				Should(ContainSubstring(
+					"Machine pool '%s' created successfully on hosted cluster '%s'",
+					npID,
+					clusterID))
+
+			By("Check the machinepool list")
+			output, err = machinePoolService.ListMachinePool(clusterID)
+			Expect(err).ToNot(HaveOccurred())
+			defer rosaClient.MachinePool.DeleteMachinePool(clusterID, npID)
```

Comment:
The delete should be on line #130, after the creation happened well

Reply:
Good catch! Updated.